src/components/inputs/Calendar.jsx

```diff
@@ -10,55 +9,51 @@ const dateFormat = 'yyyy-MM-dd';
 /**
  * Calendar component
  */
-export default class Calendar extends Component {
-  handleCalendarChange = (value) => {
+const Calendar = (props) => {
+  const {
+    id,
+    labelText,
+    helperText,
+    width,
+    maxDate,
+    minDate,
+    disableFuture,
+    disablePast,
+    selected
+  } = props;
+
+  const handleCalendarChange = (value) => {
     // Fire Dash-assigned callback
     if (value) {
-      this.props.setProps({selected: value.toISOString().split('T')[0]});
+      props.setProps({selected: value.toISOString().split('T')[0]});
```

Comment:
💡 

Maybe cleaner to unpack directly from the props object?

```js
  const {
    id,
    labelText,
    helperText,
    width,
    maxDate,
    minDate,
    disableFuture,
    disablePast,
    selected,
    setProps
  } = props;
```

Reply:
Done! 👍